### ISSUE TITLE
New API calls

### ISSUE THREAD

**Issue by FRVisser** [balena-io/balena-sdk-python#57]
Hi @nghiant2710,

The new API implementation works confusing because of the login state. If already logged in on the SDK version 1.x the old API address is retained even when updated to the latest 2.0 version. Only after a logout or removal of the resin.cfg file. But, if I try to reproduce it, I do get:
```
Settings file not found or not in proper format. Rewriting default settings to: /Users/username/.resin/resin.cfg
```

2nd point. The `device.set_to_build` command doesn't work. It returns: 
```
>>> resin.models.device.set_to_build("ef448bdb8ac0000bb9013d98c6491587","339379")
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    resin.models.device.set_to_build("ef448bdb8ac0000bb9013d98c6491587","339379")
  File "/usr/local/lib/python2.7/site-packages/resin/models/device.py", line 721, in set_to_build
    endpoint=self.settings.get('pine_endpoint')
  File "/usr/local/lib/python2.7/site-packages/resin/base_request.py", line 171, in request
    raise exceptions.RequestError(response._content)
RequestError
```

3th and last point 😅. Some return values have different labelling with API v3. 
Like:

`build` ➡️  `should_be_running__build`
`commit` ➡️ `is_on__commit`

I understand I am quite early using these commands. However, it would be nice to have some documentation on which values changed in API v3.

Thanks! 😬

**Comment by nghiant2710** [comment]
@FRVisser I'm so sorry for the late response. Regarding your questions, #2 will be fixed by PR #60 and it will be merged soon. We also update the SDK docs in #61 which shows the differences of returned object's properties between old and new API.
About the first point, the config file format of v1.7.0 and v2.0.0 will be the same so if you upgrade from v1.7.0 to v2.0.0 then your config file will be retained, it's a normal behavior. In case your config file is missing or in incorrect format (mandatory keys are not there), the SDK will create a backup file of your current config if possible and write new config file with default values.

**Comment by FRVisser** [comment]
Cool thanks!
